content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,256 @@
+This page provides a step-by-step example of updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. At the end of this tutorial, you will understand how to change the configuration for a running application. 
+This tutorial uses the NGINX image as an example.
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+## {{% heading "objectives" %}}
+* Update configuration via a ConfigMap mounted as a Volume
+* Update environment variables of a Pod via a ConfigMap 
+
+<!-- lessoncontent -->
+
+## Update configuration via a ConfigMap mounted as a Volume
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap sport --from-literal=sport=football
+```
+Below is an example of a Deployment manifest with the ConfigMap `sport` mounted as a Volume on the Pod.
+{{% code_sample file="deployments/deployment-with-configmap-as-volume.yaml" %}}
+
+Create the deployment.
+```shell
+kubectl apply -f https://k8s.io/examples/deployments/deployment-with-configmap-as-volume.yaml
+```
+
+Check the deployment.
+```shell
+kubectl get deployment configmap-volume
+```
+
+You should see an output similar to:
+```
+NAME               READY   UP-TO-DATE   AVAILABLE   AGE
+configmap-volume   3/3     3            3           19s
+```
+
+Check the pods.
+```shell
+kubectl get pods --selector=app=configmap-volume
+```
+
+You should see an output similar to:
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+configmap-volume-6b976dfdcf-qxvbm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-skpvm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-tbc6r   1/1     Running   0          72s
+```
+
+The data in the ConfigMap is mounted to the Pods as a Volume. The Pod references this data to generate an output that is printed to stdout. Check this by viewing the logs of the Pods.
+```shell
+kubectl logs <pod-name>
+```
+
+You should see the following output printed every 10 seconds:
+```
+My preferred sport is football
+```
+
+Edit the ConfigMap.
+```shell
+kubectl edit configmap sport
+```
+
+In the yaml editor, change the value of key `sport` from `football` to `cricket` and save.
+
+```yaml
+apiVersion: v1
+data:
+  sport: cricket
+kind: ConfigMap
+metadata:
+  creationTimestamp: "2023-12-27T08:24:51Z"
+  name: sport
+  namespace: default
+  resourceVersion: "1467267"
+  uid: d2d51452-8a70-40d8-b61a-0a2e92f3039b
+```
+
+You should see the following output:
+```
+configmap/sport edited
+```
+
+Tail the logs of the Pod.
+```shell
+kubectl logs -f <pod-name>
+```
+After few seconds, you should see the log output change as follows:
+```
+FROM:
+My preferred sport is football
+TO:
+My preferred sport is cricket
```

Comment:
You could try a `diff` format block. The `FROM:` and `TO:` aren't literal console output; don't make it look like that is what the reader would see.

Eg
```console
My preferred sport is football
My preferred sport is football
My preferred sport is football
My preferred sport is cricket
```

Extra credit: include a timestamp in the log lines.

Reply:
Done. It is implemented in the latest commit.